mk/spksrc.compile.mk

```diff
@@ -24,6 +24,10 @@ else
 $(POST_COMPILE_TARGET): $(COMPILE_TARGET)
 endif
 
+ifeq ($(filter $(NCPUS),0 1),)
+COMPILE_MAKE_OPTIONS += -j$(NCPUS)
+endif
+
```

Comment:
Have you tried compiling with both `PARALLEL_MAKE=X` and `-jY` flag?
That is where I foresee a collision of your added flag within `COMPILE_MAKE_OPTIONS` and resetting the make build-in `MAKEFLAGS` jobs setting.

Reply:
Yes I used the transmission package (`make -j12 TCVERSION=6.1 ARCH=x64 -C spk/transmission all-supported`). It complained about Openssl: `/spksrc/spk/transmission/work-x64-6.1/install//var/packages/transmission/target/lib/libssl.so: file not recognized: File format not recognized` Re-running the build command a few times "fixes" these errors.

Reply:
I recommend to only build without -jN for now. Maybe call it a beta feature. `make TCVERSION=6.1 ARCH=x64 -C spk/transmission all-supported` is already faster. :rocket: